### ISSUE TITLE
idea for further enhancement of program

### ISSUE THREAD

**Issue by nilax1** [globalpolicy/pubgmobilemousehelper#6]
the program itself is great but it only restricts you to use one particular gun at at a time. if you want to use another gun, one has to open the program and select another preset which is not applicable during fights. I have at least 15 presets and changing between them is difficult. my idea is that you could add another option where you can add 2 presets which could change with a single key. 

for example I have an M416 with 4x and VSS. there should be an option where I can add my preset for both these gun and change the presets when I change between them(guns). they should also be delete-able for another combination of guns.

**Comment by globalpolicy** [comment]
Well, you don't need to open the program if you want to change the active preset.
You can do that with the Enter key I believe if I recall correctly.

**Comment by nilax1** [comment]
yes but I have about 15 presets and pressing enter key 15 times doesn't seem very plausible. I will reply to your email in detail

**Comment by Schwrid** [comment]
I just want to delete the 6 built in preset that is there and I want to keep 2 only ! Can anybody help please ?

**Comment by Schwrid** [comment]
@nilax1 can you help me out dude and would be great if you sent me ur pubg Id so we could team up !

**Comment by Schwrid** [comment]
no reply from anyone :(

**Comment by nilax1** [comment]
@Schwrid sorry. I don't know anything aBOUT .NET framework

**Comment by Schwrid** [comment]
damn the creator of the program probably died :( 

RIP man :(

**Comment by Schwrid** [comment]
that nigga dead
